server/server.js

```diff
@@ -94,6 +107,19 @@ app.post(`${apiBaseUrl}/participants`,
 
     try {
       const response = await parseAndSaveParticipants(req.file.buffer);
+      const { content } = req.kauth.grant.access_token;
+      const logData = {
+        "action": "participant_post",
+        "performed_by": {
+          "username": content.preferred_username,
+          "id": content.sub
+        },
+        // Slicing to one page of results
+        "ids_posted": response.map(entry => entry.id).slice(0,10),
+        "timestamp": new Date()
+      };
+      logger.info(logData);
```

Comment:
Consider wrapping this into a function to avoid repeated code.

Reply:
done, good call